app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (sum(split_integer(6, 2)) == 6),\
+        "Sum of the parts should be equal to_value."
```

Comment:
better to use `()` then `\`
please fix everywhere

Reply:
Like that?
`def test_sum_of_the_parts_should_be_equal_to_value() -> None:
    assert (sum(split_integer(6, 2)) == 6), (
            "Sum of the parts should be equal to_value.")`

Reply:
Done.